packages/@okta/vuepress-site/docs/reference/api/policy/index.md

```diff
@@ -198,6 +198,31 @@ curl -v -X POST \
 HTTP 204:
 [Policy object](#policy-object)
 
+### Clone a Policy
+
+<ApiLifecycle access="ie" />
+
+> **Note:** This feature is only available as a part of the Identity Engine. Please [contact support](mailto:dev-inquiries@okta.com) for further information.
+
+> **Note:** Within the Identity Engine, this feature is only supported for [Authentication Policies](#authentication-policy).
+
+<ApiOperation method="post" url="/api/v1/policies/${policyId}/clone" />
+
```

Comment:
Be good to be consistent and include the Request parameters section and call out that the Policy ID is required:

#### Request parameters

The Policy ID described in the [Policy object](/docs/reference/api/policy/#policy-object) is required.